test/studies/shootout/submitted/regexredux2.chpl

```diff
@@ -26,7 +26,7 @@ proc main(args: [] string) {
   ];
 
   var data: string;
-  stdin.readstring(data); // read in the entire file
+  stdin.readAll(data); // read in the entire file
```

Comment:
Common mistake, but:  Remember that `submitted/` shootouts shouldn't be changed until the changes are reflected on the website.  The approach taken with `regexredux3.chpl` seems better if it can be applied here as well.  If it can't, we should make this a `.notest` test until the CLBG entries can be updated.

Reply:
Oops, thanks for pointing that out. I'm putting in a fix now.